specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-08-01/examples/virtualMachineExamples/VirtualMachine_Create_WithDiskControllerType.json

```diff
@@ -0,0 +1,136 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscription-id}",
+    "resourceGroupName": "myResourceGroup",
+    "vmName": "myVM",
+    "api-version": "2022-08-01",
+    "expand": "UserData"
+  },
```

Comment:
I only see the response body in the example not the request body, pls. add the request body.

Reply:
acknowledging